tests/Unit.php

```diff
@@ -2,16 +2,86 @@
 
 class Unit extends PHPUnit_Framework_TestCase {
 
-	public function subject($params = array(), $options = array()) {
-		$options += array(
-			'mock' => false,
-			'methods' => array(),
+	public static $classNamespace = "alfmarks\\";
+
+	private $subjectClass;
+	private $subjectParams = array();
+	private $subjectOptions;
+
+	private $baseParams = array();
+
+	private $baseOptions = array(
+		'mock' => false,
+		'methods' => array()
+	);
+
+	public function __construct( ...$class ) {
```

Comment:
Yeah, I like that idea.  So if there were no args passed then func_get_args would be empty and therefore would never set subjectClass.